computerGuess.js

```diff
@@ -0,0 +1,58 @@
+const readline = require('readline');
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+    return new Promise((resolve, reject) => {
+        rl.question(questionText, resolve);
+    });
+}
+
+start();
+
+async function start() {
+    console.log("THIS IS A GUESSING GAME"); //title
+    console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
+    let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
+    console.log('You entered: ' + secretNumber + '\n................ \nBelow is what I guessed \n................');
+
+//Random Number Generator function that uses (max-min)/2 for quicker answer
+function getRandom(min, max) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.round(((max - min) / 2) + min)
+}
```

Comment:
You can pull this outside of your start function so that if you needed to, you could use it throughout the rest of your code